definitions/OmnibarModeration/NewScreenedTweet.json

```diff
@@ -0,0 +1,43 @@
+{
+	"$schema": "../../schemas/event-schema.json",
+	"$meta": {
+		"vHost": "not.sure",
+		"name": "NewScreenedTweet",
+		"exchange": "moderation",
+		"routingKey": "screened.tweet",
+		"ownerTool": "OmnibarModeration"
+	},
+	"required": [
+		"message",
+		"user"
+	],
+	"properties": {
+		"message": {
+			"type": "object",
+			"required": [
+				"full_text"
+			],
+			"properties": {
+				"full_text": {
+					"type": "string",
+					"description": "Contents of the tweet."
+				}
+			},
+			"additionalProperties": false
+		},
+		"user": {
+			"type": "object",
+			"required": [
+				"name"
+			],
+			"properties": {
+				"name": {
+					"type": "string",
+					"description": "Name of the Twitter user who sent the tweet."
```

Comment:
Is this the handle or the written name on Twitter (that can be changed)? I am very sure it's the handle.